src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -191,10 +233,49 @@ def unknown_boot_partition_table_error
       #
       # @return [SetupError]
       def mbr_gap_error
-        # TRANSLATORS: error message
-        error_message = _("MBR gap size is not enough to correctly install bootloader")
+        # TRANSLATORS: error message; %s is a human readable disk size like 256 KiB
+        error_message = format(
+          _(
+            "Not enough space before the first partition to install the bootloader. " \
+            "Leave at least %s."
+          ),
+          PartitionTables::Msdos::MBR_GAP_GRUB_LIMIT
```

Comment:
This should be `PartitionTables::Msdos::MBR_GAP_GRUB_LIMIT.to_human_string` for consistency and proper i18n.